src/python/pants/engine/native.py

```diff
@@ -381,9 +381,21 @@ def extern_store_utf8(self, context_handle, utf8_ptr, utf8_len):
     c = self._ffi.from_handle(context_handle)
     return c.to_value(self._ffi.string(utf8_ptr, utf8_len).decode())
 
+  @_extern_decl('Handle', ['ExternContext*', 'uint64_t'])
+  def extern_store_u64(self, context_handle, u64):
+    """Given a context and uint64_t, return a new Handle to represent the uint64_t."""
+    c = self._ffi.from_handle(context_handle)
+    return c.to_value(u64)
+
+  @_extern_decl('Handle', ['ExternContext*', 'uint32_t'])
```

Comment:
So, because python doesn't really differentiate between these types, there isn't really much advantage to having all of these methods. You could probably drop `uint32_t` and cast on the rust side to widen things to 64.

Reply:
That's fair enough. I saw this as a tad more explicit, but it's true that it may be seen as unnecessary. I'll follow your advice.